guides/release/components-update/templates-are-html.md

```diff
@@ -0,0 +1,198 @@
+At its core, Ember is a _template_ driven framework - every part of the UI that
```

Comment:
I tried a bit of rewording:

```md
At its core, Ember's UIs are _HTML_ driven - every part of the UI that
is shown to the user is defined in an HTML template somewhere in your application.
Because of this, templates are central to Ember, and one of the most important
parts of the framework.
```